src/lib/serviceRoutines/versionTreat.cpp

```diff
@@ -79,6 +81,23 @@ std::string versionTreat
   ParseData*                 parseDataP
 )
 {
+  if (isOriginAllowedForCORS(ciP->httpHeaders.origin))
+  {
+    ciP->httpHeader.push_back(ACCESS_CONTROL_ALLOW_ORIGIN);
+    // If any origin is allowed, the header is sent always with "any" as value
```

Comment:
Comment says "any", code seems to say "*" ...
If so, please add ("*") to comment to make things clear

Reply:
Fixed in: b87cdd8